src/pagesComponents/Admin/index.jsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable no-unused-vars */
+import { Select } from 'antd';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { AiOutlineSearch } from 'react-icons/ai';
+import { setLoading } from '../../state/loading';
+import formatDate from '../../utils/formatDate';
+import {
+  AutoCompleteStyle,
+  AutoCompleteContainer,
+  TableTitle,
+  TableContent,
+  StyledTable,
+} from '../Dashboard/styles';
+// import { COMPANIES_DATA } from '../../api/endpoints';
+import Empty from '../Dashboard/Empty';
+
+export default function AdminTable() {
+  const dispatch = useDispatch();
+  const [dataSource, setDataSource] = useState([]);
+  const [filtredDataSource, setFiltredDataSource] = useState([]);
+  const [setOptions] = useState([]);
+  const [sort, setSort] = useState('');
+  const [searchValue, setSearchValue] = useState('');
+  const { loading } = useSelector((state) => state.loading.value);
+
+  const { Option } = Select;
+
+  const columns = [
+    {
+      title: <TableTitle>Company Name</TableTitle>,
+      dataIndex: 'name',
+      key: '0',
+      width: 200,
+      render: (text) => <TableContent>{text}</TableContent>,
+    },
+    {
+      title: <TableTitle>Number of employees</TableTitle>,
+      dataIndex: 'registeredEmployees',
+      key: '1',
+      width: 150,
+      render: (text) => <TableContent>{text}</TableContent>,
+    },
+    {
+      title: <TableTitle>Date of entry</TableTitle>,
+      dataIndex: 'timestamp',
+      key: '2',
+      width: 150,
+      render: (text) => <TableContent>{formatDate(text)}</TableContent>,
+    },
+    {
+      title: <TableTitle>Company ID</TableTitle>,
+      dataIndex: 'uniqueCode',
+      key: '3',
+      width: 150,
+      render: (text) => <TableContent>{text}</TableContent>,
+    },
+  ];
+
+  useEffect(() => {
+    const { CancelToken } = axios;
+    const source = CancelToken.source();
+    const fetchData = async () => {
+      dispatch(setLoading({ loading: true }));
+      try {
+        // const {
+        //   data: { data },
+        // } = await axios.get(`${COMPANIES_DATA}`);
+        dispatch(setLoading({ loading: false }));
+        // setDataSource(data);
+        setOptions(dataSource.map((company) => company.name));
+        return dataSource;
+      } catch (error) {
+        return error;
+      }
+    };
+    fetchData();
+    return () => {
+      source.cancel();
+    };
+  }, [searchValue, sort]);
+
+  const handleChange = (value) => {
```

Comment:
Why do you need another `handleChange` function that does exactly what `setSearchValue` does?